cmd/node/main.go

```diff
@@ -188,6 +188,18 @@ func loadConfig() *config.Config {
 		"do not wait for Vochain to synchronize (for testing only)")
 	flag.Int("vochainMempoolSize", 20000,
 		"vochain mempool size")
+	flag.Int("vochainSnapshotInterval", 0,
+		"create state snapshot every N blocks (0 to disable)")
+	flag.Bool("vochainStateSyncEnabled", false,
+		"during startup, let cometBFT ask peers for available snapshots and use them to bootstrap the state")
+	flag.StringSlice("vochainStateSyncRPCServers", []string{},
```

Comment:
Do we actually need to specify the peers here? Why seed nodes are not used as usual for node discovery?

Reply:
sounds like a great idea,

on closer inspection... the RPC servers required here "archive nodes", used for light client bootstrap (needed as part of state sync), are reachable over port 26657 with protocol RPC, and need to have the whole block history.
seed nodes can be "archive nodes" but not necessarily. 
even assuming seed nodes == archive nodes, i'll have to do a dirty string replace to do `s/26656/26657/` and a bit of juggling since one is a string and the other []string. but i think it's worth it :rocket: